massa-execution-worker/src/interface_impl.rs

```diff
@@ -1150,6 +1152,24 @@ impl Interface for InterfaceImpl {
 
         let mut context = context_guard!(self);
         let event = context.event_create(data, false);
```

Comment:
maybe move this line just after
```rust 
 if event_per_op >= self.config.max_event_per_operation {
    bail!("Too many event for this operation");
}
```
(only create the event if it's possible)

Reply:
Fixed